CODEOWNERS

```diff
@@ -1 +1,12 @@
-/* @defenseunicorns/uds-core @defenseunicorns/swf
+/* @defenseunicorns/uds-core @defenseunicorns/swf @defenseunicorns/zarf @defenseunicorns/pepr @defenseunicorns/lula-dev @defenseunicorns/uds @defenseunicorns/leapfrogai
+
+# Common Renovate Configuration (updates as soon as it hits `main`)
+/config/renovate.json5 @defenseunicorns/uds-core @defenseunicorns/swf @defenseunicorns/tech-leads
```

Comment:
Probably makes sense to put `@defenseunicorns/leapfrogai` here too right?

Reply:
I left it to the teams / people I know have regularly fought with renovate - we could add them but wanted to limit slightly for now.